.github/workflows/test-assistant-pr-review.yml

```diff
@@ -0,0 +1,44 @@
+name: Send PRs that require review to Slack
+
+on:
+  pull_request_target:
+    types: [review_requested]
+  issue_comment:
+    types: [created]
+
+permissions: {}
+
+jobs:
+  send-pr-to-slack-for-review:
+    runs-on: ubuntu-20.04
+    env: 
+      WOO_PR_TESTING_REVIEW_TEAM: ${{ secrets.WOO_PR_TESTING_REVIEW_TEAM }}
+    steps:
+      - name: Send ping to Slack for a new review
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify-review
+        if: |
+          github.event_name == 'pull_request' &&
+          github.event.action == 'review_requested' &&
+          github.event.requested_team.slug == env.WOO_PR_TESTING_REVIEW_TEAM
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.TEST_ASSISTANCE_BOT_TOKEN }}
+          slack-channel: ${{ secrets.WOO_PR_TESTING_REVIEW_SLACK_CHANNEL }}
+          slack-text: |
+              <!subteam^${{ secrets.WOO_PR_TESTING_REVIEW_TEAM_ID }}> was asked to review this PR:
+              <${{ github.event.pull_request.html_url }}|${{ github.event.pull_request.title }}> :thread:
+          slack-optional-unfurl_links: false
+          slack-optional-unfurl_media: false
+          
+      - name: Send ping to Slack about a new comment
+        uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d
+        id: notify-comment
+        if: github.event_name == 'issue_comment' && contains(github.event.comment.body, env.WOO_PR_TESTING_REVIEW_TEAM)
```

Comment:
Is an `issue_comment` different from a comment on a PR?  Do we want a notification if Solaris is mentioned on an issue, or if we're mentioned in a comment on a PR?  (To be honest, I don't know if there's a difference or any way to differentiate them!)

Reply:
Great question @lanej0, thanks for asking!

Internally, pull requests are a subset of issues on GitHub, as you can see [here](https://docs.github.com/en/rest/overview/issue-event-types?apiVersion=2022-11-28):

> Because pull requests are issues, issue and pull request numbers do not overlap in a repository.

There are 3 reasons why I decided to use `issue_comment` instead of `pull_request_comment`:
1. With `issue_comment`, we get more attributes we can play with in the response. For example, I wanted to use the `title` attribute in order to improve readability on Slack, but it wasn't accessible in the `pull_request_comment` event.
2. I think it's totally fine if we also get notifications on issues where the team is mentioned. The purpose is to provide a faster response by the team at every place where it's requested, so that would include issues as well.
3. The GH Actions documentation tells you to use `issue_comment` instead of `pull_request_comment` if you want to just access the PR comment details (not the PR diff comments). Considering this is our scenario, seems this is the right thing to do.